__tests__/app.test.js

```diff
@@ -50,7 +48,71 @@ describe('/api', () => {
         })
     })
 })
+describe('/api/articles', ()=>{
+    describe('GET /api/articles/:articles_id valid requests', ()=> {
+        test('200 status responds with the correct articles object', ()=>{
+            return request(app)
+            .get("/api/articles/1")
+            .expect(200)
+            .then(({body}) => {
+                const returnedObj = body
+                const expectedObj = { article :{
+                    article_id: 1,
+                    title: "Living in the shadow of a great man",
+                    topic: "mitch",
+                    author: "butter_bridge",
+                    body: "I find this existence challenging",
```

Comment:
to allow for later extension of the endpoint this  test could use a flexible matcher eg expect.objectContaining or .toMatchObject. Then you could put that created_at would be a type string - that would solve your problem :)